build-info-extractor/src/main/java/org/jfrog/build/extractor/clientConfiguration/util/DeploymentUrlUtils.java

```diff
@@ -87,14 +87,20 @@ public static String encodePath(String unescaped) {
         return escaped;
     }
 
+
     public static String buildMatrixParamsString(ArrayListMultimap<String, String> matrixParams, boolean encodeProperties)
             throws UnsupportedEncodingException {
         StringBuilder matrix = new StringBuilder();
         if (matrixParams != null && !matrixParams.isEmpty()) {
             for (String propertyKey : matrixParams.keySet()) {
-                for (String propertyValue : matrixParams.get(propertyKey)) {
-                    matrix.append(";").append(encodeProperties ? encode(propertyKey) : propertyKey)
-                            .append("=").append(encodeProperties ? encode(propertyValue) : propertyValue);
+                for (String multiPropertyValue : matrixParams.get(propertyKey)) {
+                    // Due to known bug in Artifactory, in order to support multi values properties
```

Comment:
Worth to add a reference to this bug in Artifactory.

Reply:
Hi,
I couldn't find a public open issue for that. I will contact our suitable team to make sure it will be fixed soon.